src/devices/Ssd13xx/Ssd1306.cs

```diff
@@ -50,14 +44,9 @@ private void SendCommand(ICommand command)
         {
             byte[] commandBytes = command.GetBytes();
 
-            if (commandBytes == null)
-            {
-                throw new ArgumentNullException(nameof(commandBytes));
-            }
-
-            if (commandBytes.Length == 0)
+            if (commandBytes is not { Length: >0 })
             {
-                throw new ArgumentException("The command did not contain any bytes to send.");
+                throw new ArgumentNullException(nameof(commandBytes), "Argument is either null or there were no bytes to send.");
```

Comment:
Isn't this either going to NPE on like 47 or just be an invalid value... in which case this should be `ArgumentException` *not* `ArgumentNullException`

Reply:
Good call, on both counts. Fixed.